pydgraph/txn.py

```diff
@@ -235,7 +235,7 @@ def commit(self, timeout=None, metadata=None, credentials=None):
 
         new_metadata = self._dg.add_login_metadata(metadata)
         try:
-            self._dc.commit_or_abort(self._ctx, timeout=timeout,
+            return self._dc.commit_or_abort(self._ctx, timeout=timeout,
```

Comment:
For completeness, I think the `commit_or_abort` attempt on line 246 should also return as this does (in those instances where the jwt token has expired but a re-login was successful)